reana_workflow_controller/k8s.py

```diff
@@ -179,12 +179,28 @@ def instantiate_k8s_objects(kubernetes_objects, namespace):
         current_k8s_extensions_v1beta1.create_namespaced_ingress
```

Comment:
What do you think about either renaming the function to `instantiate_chained_k8s_objects` or something similar which states this new behaviour or we make the "chaining" mechanism optional with an argument?